x/metadata/spec/02_state.md

```
@@ -43,7 +43,7 @@ Byte Array Length: `17`
 #### Scope Values
 <!-- link message: Scope -->
 
-+++ https://github.com/provenance-io/provenance/blob/v1.19.0/proto/provenance/metadata/v1/scope.proto#L70-L90
++++ https://github.com/provenance-io/provenance/blob/v1.19.0/proto/provenance/metadata/v1/scope.proto#L70-L102
```

Markdownlint

Bare URL used

(MD034, no-bare-urls)